oximeter/collector/src/bin/oximeter.rs

```diff
@@ -49,6 +49,12 @@ enum Args {
         /// The socket address at which `oximeter`'s HTTP server runs.
         #[clap(short, long, action)]
         address: SocketAddrV6,
+
+        // TODO (https://github.com/oxidecomputer/omicron/issues/4148): This flag
+        // should be removed once single node functionality is removed.
```

Comment:
I'll keep saying it: I believe we'll always want to retain the ability to run a single-node, non-replicated ClickHouse database. I'm not sure if this particular argument will change, I'm referring to the functionality itself.